<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>864</width>
    <height>481</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>871</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Введите данные о новом билете:</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_from">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>40</y>
      <width>591</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit::setValidator(QIntValidator()) </string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="placeholderText">
     <string>Откуда...</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_where">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>100</y>
      <width>591</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="placeholderText">
     <string>Куда...</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_number">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>160</y>
      <width>291</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="maxLength">
     <number>999999999</number>
    </property>
    <property name="placeholderText">
     <string>Номер рейса...</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_time">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>160</y>
      <width>291</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="placeholderText">
     <string>Время вылета...</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_date">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>220</y>
      <width>291</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="placeholderText">
     <string>Дата вылета...</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_quantity">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>220</y>
      <width>291</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="placeholderText">
     <string>Количество билетов...</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_price">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>290</y>
      <width>591</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="maxLength">
     <number>999999999</number>
    </property>
    <property name="placeholderText">
     <string>Цена...</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_return_to_adminmain">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>370</y>
      <width>291</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Вернуться в меню</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_add_ticket">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>370</y>
      <width>291</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Добавить билет</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>864</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
